.github/workflows/pr-smoke-test-play-images.yml

```diff
@@ -16,25 +16,3 @@ jobs:
       # https://github.com/playframework/playframework/pull/10819
       skip-java-17: true
       skip-java-19: true
-      skip-java-20: true
-
-  build-java-15:
```

Comment:
there is no eclipse-temurin build for Java 15 (and I think ok to just remove the Java 15 test)